baseframe/forms/validators.py

```diff
@@ -397,6 +398,21 @@ def __call__(self, form, field):
 # Legacy name
 ValidEmailDomain = ValidEmail
 
+class IsNotHttps(object):
+    """
+    Validator to confirm that a URL has 'https://'
+    """
+
+    default_message = __(u'The URL provided is not HTTPS')
+
+    def __init__(self, message=None, message_urltext=None):
+        self.message = message or self.default_message
+
+    def __call__(self, form, field):
+        if field.data:
+            if "https://" not in field.data:
+                raise ValidationError(self.message)
+
 
 class ValidUrl(object):
```

Comment:
IMO, a robust implementation will add a `schemes` parameter to ValidUrl and allow any of those schemes to pass.